odoo/addons/base/ir/ir_qweb/assetsbundle.py

```diff
@@ -307,16 +307,28 @@ def dialog_message(self, message):
             })("%s");
         """ % message.replace('"', '\\"').replace('\n', '&NewLine;')
 
-    def is_css_preprocessed(self):
+    def is_css_preprocessed(self, debug=False):
         preprocessed = True
         attachments = None
+        if debug == 'assets':
+            user_direction = self.env['res.lang'].search([('code', '=', self.env.user.lang)]).direction
+
         for atype in (SassStylesheetAsset, LessStylesheetAsset):
             outdated = False
             assets = dict((asset.html_url, asset) for asset in self.stylesheets if isinstance(asset, atype))
             if assets:
                 assets_domain = [('url', 'in', list(assets))]
                 attachments = self.env['ir.attachment'].sudo().search(assets_domain)
                 for attachment in attachments:
+                    if debug == 'assets':
```

Comment:
could you explain this part?

Reply:
@mart-e  ,
There is two problem when debug with assets, 
1) Assets bundles are not rebuilt every time so its conflict with direction when debug=assets see video for ref:  https://youtu.be/mkdUTL8fL5s
2)  If two users with different languages are request at the same time then assets-bundles create problem.